airgun/widgets.py

```diff
@@ -562,8 +562,20 @@ def messages(self):
         return result
 
     @retry_stale_element
-    def assert_no_error(self):
-        return super().assert_no_error()
+    def assert_no_error(self, ignore_messages=None):
+        if ignore_messages is None:
+            ignore_messages = []
+        self.logger.info('asserting there are no error messages')
+        for message in self.messages:
+            message_text = message.text
+            if message.type not in {'success', 'info', 'warning'}:
+                if message_text in ignore_messages:
+                    self.logger.info('ERROR MESSAGE IGNORED %s: %r', message.type, message_text)
+                    continue
+                self.logger.error('%s: %r', message.type, message_text)
+                raise AssertionError('assert_no_error: {}: {}'.format(message.type, message_text))
```

Comment:
I know this is modified version of code in widgetastic_patternfly and they aren't consistent themselves, but I would slightly prefer us to use one form of formatting inside function.

`self.logger.error('{}: {!r}'.format(message.type, message_text))` should have the same effect.